osu.Game.Rulesets.Osu/Skinning/Default/DefaultHitMarker.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osuTK;
+
+namespace osu.Game.Rulesets.Osu.Skinning.Default
+{
+    public partial class DefaultHitMarker : CompositeDrawable
+    {
+        public DefaultHitMarker(OsuAction? action)
+        {
+            var (colour, length, hasBorder) = getConfig(action);
+
+            if (hasBorder)
+            {
+                InternalChildren = new Drawable[]
+                {
+                    new Box
```

Comment:
Please don't use "x" to show a click. "x" means miss, not click.

Reply:
Not sure what other designs would be good to use. An "x" shape does a good job at being precise and big enough to be easily visible. The low-effort skinnable element I made for testing attempts to be a similar style, although I don't think it looks very good.

Reply:
I don’t think using ”x“ will lead to confusion with miss. “x” in the analysis tool is completely different from the animation with miss, it is a static sprite.

Reply:
Use a plus (+) or something.

Reply:
You could use a small circle like [MissAnalyzer does](https://imgur.com/a/Rz1vLBJ).

Reply:
The way MissAnalyzer does it won't really work here. For now, I'll change it to a plus.